### DISCUSSION TITLE
How do I execute orangepi-config?

### DISCUSSION THREAD

**Discussion by jgvega** [Joshua-Riek/ubuntu-rockchip#42] [General]
I looking for orangepi-config but I don't found. 
Is this application incorpored in this image?

I like to change the frecuency of CPU or install boot in SPI Flash with this wizard, for example.

I'm testing the version you've made and it's great. I congratulate you for this magnificent work.
To change the language to Spanish before I had to run the updates.
10 out of 10.

Greetings.

**Comment by Joshua-Riek** [comment]
The orangepi-config command does not exist currently. I will need to work on including it in the future.

But as for installing the bootloader to the SPI flash, you can do it manually from the command line. 

### Flash NVMe Bootloader to SPI
```
sudo dd if=/usr/share/orangepi/rkspi_loader.img of=/dev/mtdblock0
```

### Flash SATA Bootloader to SPI
```
sudo dd if=/usr/share/orangepi/rkspi_loader.img of=/dev/mtdblock0
echo "fdt_overlays=rk3588-ssd-sata.dtbo" | sudo tee /boot/firmware/overlays.txt
```

**Reply by jigsurfer** [reply]
Hi!

I see your reply. And i tried to follow the instruction for my fresh installation.

Hardware Orange pi 5 plus and samsung SSD 860 evo m.2

But it seem i can not get my hardwares see SSD (not nvme) from the nvme slot.

I got the same issue with stock OS from orange pi and thier instruction.
Everything work buy the SSD never appear in lsblk command, (but nvme does).

Could you please let me knoq what i did wrong. Thank you.

**Reply by shayaansiddiqui** [reply]
how would I do this if I want to use the eMMC storage?

**Reply by florian1984** [reply]
> The orangepi-config command does not exist currently. I will need to work on including it in the future.
> 
> But as for installing the bootloader to the SPI flash, you can do it manually from the command line.
> 
> ### Flash NVMe Bootloader to SPI
> ```
> sudo dd if=/usr/share/orangepi/rkspi_loader.img of=/dev/mtdblock0
> ```

Hello, on OrangePI 5 Pro there is no SPI Memory. So you can only boot from mmc/sd and copy the main System to NVM Disk. Its annoying but true :(, so orangepi-config could help for that process.
